### PR TITLE
minor fix 070 more complex parser

### PR DESCRIPTION
**Pull request by stutkhd-0709**

### CODE REVIEW

File: book/online-book/src/10-minimum-example/070-more-complex-parser.md [hunk 1]
```diff
@@ -631,6 +630,9 @@ export function baseCompile(template: string) {
 いい感じにパースができているようです。
 それではここで生成した AST を元に codegen の方の実装を進めていこうと思います。
 
+ここまでのソースコード:  
+[chibivue (GitHub)](https://github.com/Ubugeeei/chibivue/tree/main/book/impls/10_minimum_example/060_template_compiler2)
+
```
Comment (stutkhd-0709) [inline book/online-book/src/10-minimum-example/070-more-complex-parser.md:635]: Is this the correct link for the implementation?

Comment (Ubugeeei) [reply book/online-book/src/10-minimum-example/070-more-complex-parser.md:635]: Yes! Thanks!

**Review by Ubugeeei — APPROVED** [review]
Thank you 💯

File: book/online-book/src/10-minimum-example/070-more-complex-parser.md (hunk 1 above)
Comment (Ubugeeei) [reply book/online-book/src/10-minimum-example/070-more-complex-parser.md:635]: @stutkhd-0709 

Sorry, slightly incorrect x(

I'll fix them.

Comment (Ubugeeei) [reply book/online-book/src/10-minimum-example/070-more-complex-parser.md:635]: fixed in 13922cbf52c00e3012a92745b71a0507ae404c95

**Pull request merged**